suite/tests/api/ir_x86.c

```diff
@@ -787,6 +787,46 @@ test_instr_encode_and_decode(void *dc, instr_t *instr, uint len_expect,
     instr_destroy(dc, decin);
 }
 
+bool
+instr_same_synthetic(instr_t *inst1, instr_t *inst2)
+{
+    if (instr_num_srcs(inst1) != instr_num_srcs(inst2))
+        return false;
+
+    if (instr_num_dsts(inst1) != instr_num_dsts(inst2))
+        return false;
+
+    if (instr_get_isa_mode(inst1) != instr_get_isa_mode(inst2))
+        return false;
+
```

Comment:
This needs to check the operands too.  Add a TODO.

Reply:
Done.
As in: now checking the operands too.